test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -684,6 +684,11 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 				labels := node.Metadata.Labels
 				Expect(labels).To(HaveKeyWithValue("kubernetes.io/role", role))
 				Expect(labels).To(HaveKey(fmt.Sprintf("node-role.kubernetes.io/%s", role)))
+				if role == "master" && common.IsKubernetesVersionGe(
+					eng.ExpandedDefinition.Properties.OrchestratorProfile.OrchestratorVersion, "1.17.1") {
```

Comment:
Is this >= 1.17.1 correct? Shouldn't it be 1.18.0-alpha.1?

Reply:
Since we're applying the new labels to all k8s versions, it's arbitrary what version we check for here.

I chose 1.17.1 because it's not out yet, so this test would be effectively disabled until we have a new AKS Engine release that includes that minor k8s release. (I've tested it locally of course.) That would avoid the problems we had in the past with this test in our back-compat jobs.